.eslintrc.js

```diff
@@ -4,7 +4,7 @@ module.exports = {
         'node': true,
         'mocha': true
     },
-    'extends': 'airbnb',
+    'extends': 'airbnb-base',
```

Comment:
airbnb contains Reactjs rules by default - thats why we had lots of dependencies in packagejs - changed that